src/App.js

```diff
@@ -1,7 +1,7 @@
 import React, { useState, useEffect } from 'react';
-import Slider from './components/Slider';
+import Slider from './Components/Slider';
```

Comment:
line 2 in your App.js: import Slider from './Components/Slider';
components needs to be lowercase